tests/pytorch/test_stream.py

```diff
@@ -24,7 +53,6 @@ def test_basics():
     s = torch.cuda.Stream(device=F.ctx())
     with torch.cuda.stream(s):
         xx = x.to(device=F.ctx(), non_blocking=True)
-    with FS.stream(s):
         gg = g.to(device=F.ctx())
     s.synchronize()
     OPS.copy_u_sum(gg, xx)
```

Comment:
I think we need correctness check here as well, sth like: `assert (OPS.copy_u_sum(gg, xx) == OPS.copy_u_sum(g, x))`

Reply:
Updated accordingly.